static/cfg-view.js

```diff
@@ -156,6 +157,16 @@ define(function (require) {
             this.saveState();
         }
     };
+    
+    Cfg.prototype.closeCfg = function () {
+         _.defer(function (self) {
+                self.container.close();
+            }, this);
+    };
+    
+    Cfg.prototype.onBinaryMode = function() {
+        this.closeCfg();
```

Comment:
I don't think closing the cfg is the way to go here. Maybe update the graph to only say "CFG not supported on binary mode" liek we do when no output is found

Reply:
right! it's a better idea!